apollo-router/src/uplink/license_enforcement.rs

```diff
@@ -183,17 +216,47 @@ impl LicenseEnforcementReport {
                 .build(),
         ]
     }
+
+    fn schema_restrictions() -> Vec<SchemaRestriction> {
+        vec![
+            SchemaRestriction::builder()
+                .name("@authenticated")
+                .url("https://specs.apollo.dev/authenticated/v0.1")
+                .build(),
+            SchemaRestriction::builder()
+                .name("@requiresScopes")
+                .url("https://specs.apollo.dev/requiresScopes/v0.1")
```

Comment:
const ?

Reply:
moved to https://github.com/apollographql/router/pull/3949